### DISCUSSION TITLE
MultiSelect

### DISCUSSION THREAD

**Discussion by charlie17** [skeletonlabs/skeleton#741] [Q&A] [answered]
Just checking out Skeleton. Impresssive stuff. Is a multiselect component available in some way or on the roadmap?

Have found this to be a key thing for data entry / option handling.

As in the following:

https://mantine.dev/core/multi-select/

**Comment by endigo9740** [comment, marked as answer]
This will introduce something along those lines:
https://github.com/skeletonlabs/skeleton/issues/375
